### ISSUE TITLE
Orbital hypoplasia

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11722]
**Preferred term label:**
Orbital hypoplasia

**Synonyms**
Underdevelopment of the bony eye socket

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C2013762

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/clinicalSynopsis/621650

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001547
- label: Orbital hypoplasia
- parents: [Abnormal morphology of bony orbit of skull (HP:3000030)](https://hpo.jax.org/app/browse/term/HP:3000030)
- pmid: 
- def: Underdevelopment of the bony eye socket.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11722
- synonyms:
